data/metrics_definitions.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
```

Comment:
Is this nested object structure necessary? I think we can totally go with just key-value pairs like 
```json 
{ 
   "cer": { ... }
}

Reply:
Maybe I don't get it, but isn't it the same as what I did (except that I have one more newline)?

Reply:
I mean like: 
```json 
{ 
   "cer": { ... },
   "wall_time": { ... },
   "cpu_time": { ... }
}

Reply:
Done: 0e0277825f654f582acb979e1f797436d250e572